package.json

```diff
@@ -33,12 +33,12 @@
     "eslint": "6.8.0",
     "eslint-config-prettier": "6.10.0",
     "eslint-plugin-import": "2.20.1",
-    "eslint-plugin-prettier": "^3.1.2",
+    "eslint-plugin-prettier": "3.1.2",
     "eslint-plugin-react": "7.19.0",
     "ionicons": "5.0.1",
     "prettier": "1.19.1",
-    "react": "16.13.0",
-    "react-dom": "16.13.0",
+    "react": "16.13.1",
```

Comment:
Not sure we should start using deps and devDeps. Wouldn't react be a dep as well? I would move everything to `dependencies`.

Reply:
done